ansible-scylla-node/tasks/common.yml

```diff
@@ -299,6 +299,63 @@
       when: token_distributor is defined
   when: start_scylla_service is defined and start_scylla_service|bool
 
+- name: Check if authentication is enabled
+  lineinfile:
+    path: /etc/scylla/scylla.yaml
+    regexp: '^(authenticator: (.*PasswordAuthenticator.*|.*TransitionalAuthenticator.*))|(authorizer: (.*CassandraAuthorizer.*|.*TransitionalAuthorizer.*))'
+    state: absent
+  check_mode: yes
+  register: _authentication_enabled
+  run_once: true
+
+- name: If authentication is enabled, adjust replication for system_auth keyspace
+  block:
+    - fail:
+        msg: "Unable to adjust system_auth replication unless start_scylla_service is set to True"
+      when: start_scylla_service == False
```

Comment:
What if you were not supposed to start the service but it was already running?
You should check if `scylla` is running instead.

Reply:
Fixed